tests_e2e/tests/lib/virtual_machine_client.py

```diff
@@ -166,7 +166,7 @@ def restart(
                 # that the reboot actually happened by checking the system's uptime.
                 log.info("Verifying VM's uptime to ensure the reboot has completed...")
                 try:
-                    uptime = ssh_client.run_command("cat /proc/uptime | sed 's/ .*//'", attempts=1).rstrip()  # The uptime is the first field in the file
+                    uptime = ssh_client.run_command("cat /proc/uptime | sed 's/ .*//'", attempts=3).rstrip()  # The uptime is the first field in the file
```

Comment:
this code has its own retry logic, that is why it sets attempts to 1

you should add the new error to the except below, on line 176